quickwit-indexing/src/actors/indexing_pipeline.rs

```diff
@@ -439,19 +441,17 @@ impl IndexingPipeline {
         if self.handlers.is_some() {
             return Ok(());
         }
+        self.previous_generations_statistics.num_spawn_attempts = 1 + retry_count;
         if let Err(spawn_error) = self.spawn_pipeline(ctx).await {
-            if let Some(duration_before_retry) = Self::wait_duration_before_retry(retry_count) {
-                error!(err=?spawn_error, "Error while spawn_pipeline, waiting 1000ms");
-                ctx.schedule_self_msg(
-                    duration_before_retry,
-                    IndexingPipelineMessage::Spawn {
-                        retry_count: retry_count + 1,
-                    },
-                )
-                .await;
-            } else {
-                return Err(ActorExitStatus::Failure(Arc::new(spawn_error)));
-            }
+            let duration_before_retry = Self::wait_duration_before_retry(retry_count);
+            error!(err=?spawn_error, "Error while spawn_pipeline, waiting 1000ms");
```

Comment:
The error message is inaccurate. I'd go with something like:
```
 error!(err=?spawn_error, retry_count = .., retry_delay = .., "Error while spawning indexing pipeline, retrying after some time.");
 ```

Reply:
my bad, I should have cleaned that too :/